.github/pull_request_template.md

```diff
@@ -1,18 +1,18 @@
-Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.
+Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   
 
 Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.
 
 - [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
 - [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
-- [ ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html)
+- [ ] Updated relevant unit tests or have written new unit tests, which should include edge cases, e.g. empty datasets, errors, etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
```

Comment:
The Unit Test guide doesn't mention any "edge cases", what exactly is an edge case? and should it be included in here?

Reply:
Maybe I should use https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios ?

Reply:
I think the entire document is relevant so I'm not sure I would point to a specific section of the unit test guide, but are there any documentations that provide further details about edge cases, if not I think pointing to the unit guide might be better..

@bms63 thoughts?

Reply:
I don't think we want to get to in the weeds with edge cases in the PR Template, but we could expand on them in the guidance.   For example, edge case might be NAs, partial data, points almost at the <= or >= cut-offs, weird characters that can pop up in data we deal with.  Perhaps a few examples from the test files would suffice??

I didn't realize we had a section on realistic data scenarios.

Reply:
I think we wanted to prompt people to think about edge cases in teh PR templates - as this has been some incidents where we could of caught bugs earilier if we had been more thoughtful in our edge cases.

Reply:
Looks like I may have pulled the trigger a little too quick there for another commit. I also thought it was checkbox with the link was more as a reminder and once you comb over the unit-test doc a couple times it feels clear to try and come up with "weird" test cases

Reply:
Should I just add an additional blurb on the guidance for this PR as well or make it a separate issue? @bms63